infrastructure/demo.tfvars

```diff
@@ -8,3 +8,5 @@ feature_admissions = "true"
 capacity = "2"
 feature_paid_in_full =  "true"
 website_local_cache_sizeinmb = "0"
+log_level = "DEBUG"
+node_debug = "child_process,cluster"
```

Comment:
locally - I dont see any difference.... so im hoping when we crash and burn this will spill out with useful information. Other options look way too verbose:
`timer
http
net
fs
tls
stream
module`

Reply:
Also I dont think setting the LOG_LEVEL on our application to DEBUG will help much.